frontend/src/App.js

```diff
@@ -246,7 +246,7 @@ const RouteList = () => {
       <Route path="/privacy-policy" element={<Privacy />} />
       <Route exact path="/coming-soon" element={<div />} />
       <Route exact path="/not-found" element={<div />} />
-      <Route path="*" element={<Navigate replace to="/not-found" />} />
```

Comment:
Hi @abhishek-p-s , The not-found page. We couldn't see it in this example link: http://localhost:3000/test

Reply:
sorry i commented that for debuging the route. reverted and pushed again.